src/wazuh_modules/inventory_harvester/tests/CMakeLists.txt

```diff
@@ -2,20 +2,14 @@ cmake_minimum_required(VERSION 3.12.4)
 
 project(harvester_tests)
 
-# include_directories(${CMAKE_SOURCE_DIR})
-# include_directories(${CMAKE_SOURCE_DIR}/include/)
+set(CMAKE_CXX_FLAGS_DEBUG "-g --coverage -fsanitize=address,leak,undefined")
+
 include_directories(mocks/)
```

Comment:
I think this is redundant if we use set(TEST_MOCK_DIR ${CMAKE_CURRENT_LIST_DIR}/mocks)

Reply:
It is, I'll remove it.